### PR TITLE
fix: Fix targets handling in Babel preset

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: packages/babel-preset/index.js [hunk 1]
```diff
@@ -37,8 +38,9 @@ const defaultOptions = {
   runtime: false,
   corejs: false,
   debug: false,
+  targets: undefined, // Targets for @babel/preset-env.
   ignoreBrowserslistConfig: false,
-  browsers: defaultBrowsers,
```
Comment (taion) [inline packages/babel-preset/index.js]: this wasn't doing anything

File: packages/babel-preset/index.js [hunk 2]
```diff
@@ -37,8 +38,9 @@ const defaultOptions = {
   runtime: false,
   corejs: false,
   debug: false,
+  targets: undefined, // Targets for @babel/preset-env.
```
Comment (taion) [inline packages/babel-preset/index.js]: might as well document things a bit?

File: packages/babel-preset/index.js [hunk 3]
```diff
@@ -37,8 +38,9 @@ const defaultOptions = {
...
   ignoreBrowserslistConfig: false,
-  browsers: defaultBrowsers,
+  configPath: '.',
```
Comment (taion) [inline packages/babel-preset/index.js]: seems cleaner to put the default here

File: packages/babel-preset/index.js [hunk 4]
```diff
@@ -53,21 +55,27 @@ function getTargets(
...
-  // TODO: distinguish between app and libraries for node as well
+
+  // TODO: Distinguish between app and libraries for node as well.
   if (target === 'node') return { node: '10.0' };
+
   if (
     ignoreBrowserslistConfig ||
-    !loadConfig({ path: configPath || path.resolve('.') })
-  )
-    return { browsers: targets.browsers || defaultBrowsers };
```
Comment (taion) [inline packages/babel-preset/index.js]: `targets.browsers` is going away eventually. also `targets` might be null-y here

File: packages/babel-preset/index.js [hunk 5]
```diff
@@ -53,21 +55,27 @@ function getTargets(
...
+  // We don't run browserslist ourself b/c that would require doing a bunch
+  // of additional transforms to get the output in a format preset-env would
+  // accept.
   return targets || undefined;
 }
 
-function preset(_, explicitOptions = {}) {
-  const env = _.env() || 'production'; // default to prod
+function preset(api, explicitOptions = {}) {
+  // FIXME: This is meant to default to production, but Babel actually defaults
```
Comment (taion) [inline packages/babel-preset/index.js]: ARGH... we're actually defaulting to `development` because that's what `api.env()` does

**Comment by jquense** [comment]
which bit is the fix?

File: packages/babel-preset/index.js (hunk 5 above)
Comment (jquense) [reply packages/babel-preset/index.js]: eww

**Comment by taion** [comment]
right sorry. it's essentially here: https://github.com/4Catalyzer/javascript/pull/68/files#diff-122f9dd69d07e88ae976a0dd495763deL62

if you don't pass in `targets`, then this just crashes on trying to dereference `targets.browsers`. but since `targets.browsers` is deprecated anyway, _and_ we don't use it AFAICT, then we may as well drop it?

**Pull request merged**